services/tns_submission_queue/tns_submission_queue.py

```diff
@@ -393,15 +427,30 @@ def build_reporters_string(submission_request, source, tnsrobot, session):
             affiliations = ", ".join(affiliations)
             reporters.append(f'{author.first_name} {author.last_name} ({affiliations})')
 
+            if author.is_bot:
+                # if the author is a bot, we use the bio as the remark
+                bio = str(author.bio).strip()
+                # if it doesn't end in a period, add one
+                if bio[-1] != ".":
```

Comment:
Would maybe also be good to enforce this when folks write and submit the description too.

Reply:
Will do!